src/plugins/foxPage/components/Layout.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Text } from '@chakra-ui/react'
+import { ReactNode } from 'react'
+import foxPageBg from 'assets/foxpage-bg.png'
+import { AssetIcon } from 'components/AssetIcon'
+
+type FoxLayoutProps = {
+  content: ReactNode
+  icon: string
+  title: string
+  description: string
+}
+
+export const Layout = ({ content, icon, title, description }: FoxLayoutProps) => {
+  return (
```

Comment:
As we've discussed, the API is very early for now and will most likely need a selected asset (ID) in the future, as well as a hook for getting all the opportunities data to be passed to children, but that's for wire-up, these components are purely presentational for now.